### ISSUE TITLE
Items import needed

### ISSUE THREAD

**Issue by joligario** [EQEmu/Server#3701]
The content database is way behind live on items. The 13th floor import script is ready to go live. Need some eyes on the data as of 20 NOV to see if any last changes need to happen before importing. Please review and provide any comments if you feel we are ready to make the updates.

[items_20231120.zip](https://github.com/EQEmu/Server/files/13421147/items_20231120.zip)
[items_new_20231120.zip](https://github.com/EQEmu/Server/files/13421148/items_new_20231120.zip)

**Comment by fryguy503** [comment]
All of those mod2's dropped to 0 are because they were all moved to heroic stats

**Comment by joligario** [comment]
Which mod2s? I want to make sure I annotate correctly.

**Comment by joligario** [comment]
My review complete and incorporated above. Take a look at what you can and let me know what you think. We can leave alone, merge, change completely, or a combination in the script. I'll mark with a comment on the plan above as I receive feedback.

**Comment by Aeadoin** [comment]
> All of those mod2's dropped to 0 are because they were all moved to heroic stats
> 
> So if you update heroic stats but not the mod2's you will double the performance of those stats.

Correct, we don't have the Mod2 to Heroic Stat functionality implemented (I believe this wasn't actually done until a later client), So I would recommend not changing the value of any mod2s.

**Comment by fryguy503** [comment]
> > All of those mod2's dropped to 0 are because they were all moved to heroic stats
> > So if you update heroic stats but not the mod2's you will double the performance of those stats.
> 
> Correct, we don't have the Mod2 to Heroic Stat functionality implemented (I believe this wasn't actually done until a later client), So I would recommend not changing the value of any mod2s.

Except the source does calculate them:

https://github.com/EQEmu/Server/blob/b03f52de18495ee80ffb80d773ede41d3fbbe86f/zone/bonuses.cpp#L5923-L5948

https://github.com/EQEmu/Server/blob/b03f52de18495ee80ffb80d773ede41d3fbbe86f/zone/bonuses.cpp#L5989-L6000

**Comment by fryguy503** [comment]
Accuracy: Checks against NPC Avoidance to determine if you hit. (Dexterity)

Avoidance: Checks against an NPCs Accuracy to determine if you're hit. (Agility)

Combat Effects: Increases your chance to proc with all skills, abilities and weapons. (Dexterity)

Damage Shielding: Increases YOUR Damage Shield damage. (Strength)

Damage Shield Mitigation: Decreases damage sustained from enemy Damage Shields. (Strength)

DoT Shielding: Decreases DoT damage sustained. (Intelligence or Wisdom)

Melee Shielding: Mitigates incoming melee damage by a %. (Stamina)

Spell Shielding: Mitigates incoming spell damage by a %. (Intelligence or Wisdom)

Strike Through: Checks against an enemies Parry/Riposte/Dodge. Higher Strike Through gives you a chance to win the roll and punch them in the face anyway. (Agility)

Stun Resistance: Gives you a fixed percent chance to not be stunned (Stamina)


Yea I just looked at Agility and saw the avoidance in there.

**Comment by fryguy503** [comment]
IMO we shouldn't be in between on the items, it should be all pre heroic stat changes or post heroic stat change.

If we are going to import all new item data, we should add the heroic stat modifiers prior and import all item data as it is.
